services/approved_studies.py

```
@@ -352,6 +352,13 @@ async def update_approved_studies_landscape(
                     user_data["drug_combination_regimen_root_id"] = (
                         brand_rel.parent_root_id
                     )
+            elif (len(brand_relationships_data) == 0) and not (
```

Ruff (0.8.2)

Undefined name `brand_relationships_data`

(F821)